assets/src/edit-story/utils/debounce.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Credit: https://davidwalsh.name/javascript-debounce-function
```

Comment:
Need to be careful including code from third-party sites due to license. Although gladly this one's under MIT license.

Don't think we need that function here anyway though, see comment above re: `useDebouncedCallback`

Reply:
I did check license - it's just extracted from Underscore, which is MIT.